pgjdbc/src/test/java/org/postgresql/test/jdbc2/PreparedStatementTest.java

```diff
@@ -542,9 +542,9 @@ public void testSetFloatInteger() throws SQLException {
     assertTrue(rs.next());
 
     assertTrue("expected " + maxFloat + " ,received " + rs.getObject(1),
-        ((Double) rs.getObject(1)).equals(maxFloat));
+        rs.getObject(1).equals(maxFloat));
```

Comment:
If you touch those kind of lines, `assertEquals` would be better (it would output error message in IDE-compatible format)